couchbase-entity-api/src/main/java/kaufland/com/coachbasebinderapi/schema/DefaultSchemaValidator.kt

```diff
@@ -0,0 +1,57 @@
+package kaufland.com.coachbasebinderapi.schema
+
+open class DefaultSchemaValidator : SchemaValidator {
+
+    override fun validate(current: List<EntitySchema>, released: List<EntitySchema>, logger: SchemaValidationLogger) {
+        val currentMap: Map<String, EntitySchema> = current.map { it.name to it }.toMap()
+        val releasedMap: Map<String, EntitySchema> = released.map { it.name to it }.toMap()
```

Comment:
`.associateBy { it.name }` is what you're looking for: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.sequences/associate-by.html